docs/source/index.md

```diff
@@ -1,10 +1,47 @@
-# arviz-stats
-ArviZ statistics and diagnostics functions.
+# ArviZ-stats
+
+
+Welcome to the ArviZ-stats documentation! This library focuses on statistical functions and diagnostics for exploratory analysis of Bayesian models. It is one of the 3 components of the ArviZ library, the other two being:
+
+* [arviz-base](https://arviz-base.readthedocs.io/en/latest/) data related functionality, including converters from different PPLs.
+* [arviz-plots](https://arviz-plots.readthedocs.io/en/latest/) for visual checks built on top of arviz-stats and arviz-base.
+
+We recommend most users install and use all three ArviZ components together through the main ArviZ package, as they're designed to work seamlessly as one toolkit. Advanced users may choose to install components individually for finer control over dependencies.
+
+Note: All statistical functions - whether accessed through the full ArviZ package or directly via ArviZ-stats - are documented here.
+
+
+## Overview
+
+In Modern Bayesian statistics models are usually build and solve using probabilistic programming languages (PPLs) such as PyMC, Stan, NumPyro, etc. These languages allow users to specify models in a high-level language and perform inference using state-of-the-art algorithms like Markov Chain Monte Carlo (MCMC) or Variational Inference (VI). As a result we usually get a posterior distribution, in the form of samples. The posterior distribution has a central role in Bayesian statistics, but other distributions like the posterior and prior predictive distribution are also of interest. And other quantities may be relevant too.
+
+The correct visualization, analysis, and interpretation of these computed data is key to properly answer the questions that motivate our analysis.
+
+When working with Bayesian models there are a series of related tasks that need to be addressed besides inference itself:
+
+* Diagnoses of the quality of the inference
+
+* Model criticism, including evaluations of both model assumptions and model predictions
+
+* Comparison of models, including model selection or model averaging
+
+* Preparation of the results for a particular audience.
+
+We call these tasks exploratory analysis of Bayesian models (EABM). Successfully performing such tasks are central to the iterative and interactive modelling process (See Bayesian Workflow). In the words of Persi Diaconis.
```

Comment:
I think the "(see Bayesian Workflow)" part was intended to include a link to https://arviz-devs.github.io/EABM/Chapters/Bayesian_workflow.html